packages/ui/components/ui/block-sync-status/large.tsx

```diff
@@ -12,7 +12,9 @@ export const LargeBlockSyncStatus = ({
   error,
 }: BlockSyncProps) => {
   if (error) return <BlockSyncErrorState error={error} />;
-  if (!latestKnownBlockHeight || !fullSyncHeight) return <div />;
+  if (!latestKnownBlockHeight || !fullSyncHeight) {
```

Comment:
Shouldn't this be checking for `!== undefined`? Since `0n` now means that the first block has synced?

Reply:
Aha! This edge case is the gift that keeps on giving.